test/include/autopart_test.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require "yaml"
+require "yast"
+
+Yast.import "Profile"
+Yast.import "Arch"
+Yast.import "Partitions"
+
+describe "Yast::AutoinstallAutopartInclude" do
+  FIXTURES_PATH = File.join(File.dirname(__FILE__), '../fixtures')
+
+  module DummyYast
+    class AutoinstallAutopartClient < Yast::Client
+      def main
+        Yast.include self, "autoinstall/autopart.rb"
+        @planHasBoot=false
+      end
+
+      def initialize
+        main
+      end
+      def planHasBoot
+        @planHasBoot
+      end
+      def planHasBoot=(v)
+        @planHasBoot = v
+      end     
+    end
+  end
+
+  subject(:client) { DummyYast::AutoinstallAutopartClient.new }
+
+  describe "#autopart" do
+
+    describe "ppc partitioning" do   
+      let(:target_map) { YAML.load_file(File.join(FIXTURES_PATH, 'storage', 'target_ppc.yml')) }
+      let(:ay_device) { Yast::Profile.current['partitioning'].find {|x| x["device"] == checked_device} }
+      before do
+        allow(Yast::Arch).to receive(:ppc).and_return(true)
+        allow(Yast::Partitions).to receive(:MinimalNeededBootsize).and_return(800000)
+        allow(Yast::Partitions).to receive(:DefaultBootFs).and_return(:ext4)
+        allow(Yast::Partitions).to receive(:FsidBoot).and_return(Yast::Partitions.fsid_gpt_prep)
+        Yast::Profile.ReadXML(File.join(FIXTURES_PATH, 'profiles', 'ppc_partitions.xml'))
+      end
+
+      context "AY configuration file has no /boot partition" do
+        before do
+          client.planHasBoot=false
```

Comment:
Yes, but unfortunately this flag wil be used for /boot and prep partitions. But at least I have changed the naming.